content/en/opentelemetry/collector_exporter/management.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Do we actually need this page?
This is great, but it is not related to Datadog, and we are not teaching the user how to do it.
They still need to go to https://opentelemetry.io/docs/collector/management/#opamp in order to learn more.

I'd get rid of this section.

Reply:
Sounds good. I'll remove this page and instead put a link to the OTel Management docs in the Configuration further reading. 👍